crates/ty_ide/src/goto.rs

```diff
@@ -599,6 +599,7 @@ impl GotoTarget<'_> {
                     // Find the containing call expression from the ancestor chain
                     let call_expression = covering_node
                         .ancestors()
+                        .rev()
```

Comment:
I think we should fix this in `ancestors` instead. At least one of the remaining references are equally wrong and we can call `rev` for the remaining usages where they want to walk the ancestors top to bottom (but bottom up is more intuitive to me)